.gitignore

```diff
@@ -1,4 +1,5 @@
 dist
+espIdfDocsIndex_lang_*.json
```

Comment:
If we close the extension (or vscode) then what happens is that, we will try to download the index again for the same version... IDK why maybe since we are using the `tmpdir()`?

Reply:
No sorry, was using it before, but I've figured for each index there is going to be a file which still need to be updated. Decided to have the index as variable in memory `src/config.ts` and change when `idf.espIdfPath` changes.

Will remove from `.gitignore`

Reply:
> but I've figured for each index there is going to be a file which still need to be updated

true but a version's doc shouldn't change right, doc is updated only when a new version is released, right?

If so then we can just simply cache the doc with filename based on the URL

Reply:
Versions itself should not change but `latest` will so for that case we could have some timestamp for each day?